docs/project_info/contributing.rst

```diff
@@ -97,26 +97,98 @@ Testing Guidelines
 
 To ensure that you do not introduce bugs into Snakemake, you should test your code thoroughly.
 
-To have integration tests run automatically when committing code changes to Github, you need to sign up on wercker.com and register a user.
+Putting these tests into repeatable test cases ensures they can be checked on multiple platforms and Python versions.
+
+Setup to run the test suite locally
+===================================
+
+Unit tests and regression tests are written to be run by `pytest <https://docs.pytest.org/en/stable/>`_.
+
+Assuming you are on a Linux system, and have a working Conda installation, the easiest way to run the tests is like so:
+
+1. Ensure your Conda config (``~/.condarc``) has these options:
+
+.. code-block::
+
+   channels:
+     - conda-forge
+     - bioconda
+   channel_priority: strict
+   solver: libmamba
```

Comment:
The channel setup is not needed for the tests to work. They all come with their own conda envs, don't they? Also the libmamba setting can be avoided by just requiring a minimum conda version (24.7.1) to be used (which uses libmamba by default). I think that is prefereable, since the libmamba integration in earlier conda versions was problematic.

Reply:
Hence, I would suggest to just say here that conda 24.7.1 should be present and strict priorities (linking to https://conda-forge.org/docs/user/tipsandtricks/#using-multiple-channels) and suggesting to run the command `conda config --set channel_priority strict` should be sufficient here.